firebase/functions/src/index.ts

```diff
@@ -355,12 +355,14 @@ exports.addProjectTopicKey = functions.https.onRequest(async (_, res) => {
             res.status(404).send('No projects found');
         }
 
-        if (!isEmptyProject && data) {
+        if (data) {
```

Comment:
Any reason this check was removed?

Reply:
we don't need to check here because project  empty  condition is already handled above.